src/instrumentation.ts

```diff
@@ -54,13 +54,13 @@ export class AzureFunctionsInstrumentation extends InstrumentationBase {
             azFunc.app.hook.preInvocation((context) => {
                 const traceContext = context.invocationContext.traceContext;
                 if (traceContext) {
-                    context.functionHandler = otelContext.bind(
-                        propagation.extract(otelContext.active(), {
-                            traceparent: traceContext.traceParent,
-                            tracestate: traceContext.traceState,
-                        }),
-                        context.functionHandler
-                    );
+                    const extractedContext = propagation.extract(otelContext.active(), {
+                        traceparent: traceContext.traceParent || traceContext.traceParent,
+                        tracestate: traceContext.traceState || traceContext.traceState,
```

Comment:
Is traceparent and tracestate case insensitive? In python I have it in all lower case. 
If the case matters, this could be the reason traces are not propogating.
We might need to do something like

`    traceparent: traceContext.traceParent ?? traceContext.traceparent,
`

Reply:
Yes typescript is tcase sensitive.

Reply:
case sensitive?